src/main/resources/static/api.yml

```diff
@@ -3243,6 +3243,12 @@ components:
         assessmentId:
           type: object
           $ref: '#/components/schemas/OASysAssessmentId'
+        assessmentState:
+          $ref: '#/components/schemas/OASysAssessmentState'
+        date:
+          type: string
+          format: date-time
+          description: If the assessment is completed this is the date it was completed, otherwise it is the date it was initiated
```

Comment:
hmm, on consideration I'm not sure this is actually very helpful...

Perhaps it would be clearer to always return both:

- `dateStarted: date`, and
- `dateCompleted: date or null`

That way the UI can display the date information more flexibly?

Reply:
Have updated 👍 will follow up with PR for same change to offence analysis endpoint